### ISSUE TITLE
developmental math: tokens and token lists 

### ISSUE THREAD

**Issue by lwtchu** [openstax/cnx-recipes#2186]
<img width="743" alt="Screen Shot 2020-04-09 at 10 38 29 AM" src="https://user-images.githubusercontent.com/11299125/78916740-90e4a800-7a53-11ea-8de3-50e42b138014.png">

Tokens and token lists are comprised of a few things
- stylized tokens for the ⓐ, ⓑ, ⓒ etc characters
- a list class that calculates the amount of space for the list and if each item can be in line, it is in line 

## Stylized tokens specs
This is targeted by `<span class="token>ⓐ</span>`.
Color: B2:   `#1E7E91`
Size: font-scale(3): 14.25pt, 19.01px, 1.7rem
Font: Stix (the one we are using for math)

## Token list 
Targetd by `<list list-type="enumerated" number-style="arabic" class="circled">`

- Remove the enumerated numbers
- The circled class should trigger the mechanism that tries to align horizontally short items side by side. 

Example
1.77 and 1.78 are too long so they are stacked like a regular list
1.81-1.84 are shorter, so they are aligned on the same line.
<img width="868" alt="Screen Shot 2020-04-09 at 2 14 35 PM" src="https://user-images.githubusercontent.com/11299125/78932075-82ef5100-7a6c-11ea-8421-16bfcde42879.png">

**Comment by wilkus27** [comment]
@lwtchu How we should style Stepwise lists? In prealgebra we can find them 

- in Key Concepts:
![image](https://user-images.githubusercontent.com/40228854/85142688-08e7ff00-b249-11ea-99ea-7638c43ce602.png)

and in How To notes:
![image](https://user-images.githubusercontent.com/40228854/85142765-22894680-b249-11ea-93d6-01f7cb1a4b7a.png)

**Comment by lwtchu** [comment]
@wilkus27 i'll make a separate issue for the stepwise lists

**Comment by brittweinstein** [comment]
This requires the STIX font to work -- we are waiting for this to be on staging before we can work on this.
